eradiate/scenes/measure.py

```diff
@@ -443,6 +443,22 @@ class PerspectiveCameraMeasure(Measure):
         validator=validator_is_positive
     )
 
+    @target.validator
+    @origin.validator
+    def _target_origin_validator(self, attribute, value):
+        if np.all(self.target == self.origin):
```

Comment:
Strictly speaking yes: The tolerance for `allclose()` is non-zero but even a tiny difference would allow for a valid specification of a `look_at` transform.

Reply:
I'll then just commit this suggestion out of consistency with regular floating point arithmetics practices. Let's keep our eyes open, we'll change it to strict equality if it proves to be necessary.